generate_gun_cmd.py

```diff
@@ -0,0 +1,247 @@
+"""
+@file generate_gun_cmd.py
+@author Ryan Missel
+
+Command line based script for users to generate guns and get a PDF of the BnB Card Design
+filled out by the gun attributes that are generated.
+
+Takes in user-input on specific gun features to choose
+"""
+import os
+import fitz
+import pdfrw
+import locale
+import argparse
+import subprocess
+
+from pdfrw import PdfReader
+from classes.Gun import Gun
+from classes.GunImage import GunImage
+
+# KEY Names for PDF
```

Comment:
Wow you went hard on PDFs here. I am going to skip this for review

Reply:
Misery